libs/ui-toolkit/src/components/tooltip/tooltip.tsx

```diff
@@ -58,7 +58,7 @@ export const Tooltip = ({
     children
   );
 
-export const TooltipCellComponent = (props: ITooltipParams) => {
+export const AgTooltip = (props: AgTooltipProps) => {
```

Comment:
chore (non blocking): I don't think we should be mixing our uitoolkit tooltip with something more specific to ag grid